doc/Makefile

```diff
@@ -14,9 +14,10 @@ SPHINXPROJ    = "Project ACRN"
 SOURCEDIR     = .
 BUILDDIR      = _build
 
+ACRN_BASE     = "`pwd`/../.."
```

Comment:
Could we use `ACRN_BASE = $(CURDIR)/../..` here instead? (or `$(shell pwd)/../..`)